src/ocnews_api.cpp

```diff
@@ -206,6 +206,14 @@ rsspp::feed ocnews_api::fetch_feed(const std::string& feed_id) {
 		json_object_object_get_ex(item_j, "body", &node);
 		item.content_encoded = json_object_get_string(node);
 
+		json_object_object_get_ex(item_j, "enclosureMime", &node);
+		if (node)
+			item.enclosure_type = json_object_get_string(node);
+
+		json_object_object_get_ex(item_j, "enclosureLink", &node);
+		if (node)
+			item.enclosure_url = json_object_get_string(node);
+
```

Comment:
This slightly differs from what both [Atom](https://github.com/newsboat/newsboat/blob/master/rss/atom_parser.cpp#L104) and [RSS 0.9x](https://github.com/newsboat/newsboat/blob/master/rss/rss_09x_parser.cpp#L105) parsers do, because:
1. it might initialize just one of the fields;
2. it will initialize the fields even if MIME isn't accepted by `utils::is_valid_podcast_type`.

Can you please change it like so?
```c++
{
    json_object* type_obj;
    json_object_object_get_ex(item_j, "enclosureMime", &type_obj);
    json_object_object_get_ex(item_j, "enclosureLink", &node);
    if (type_obj && node) {
        const std::string type = json_object_get_string(type_obj);
        if (utils::is_valid_podcast_type(type)) {
            item.enclosure_url = json_object_get_string(node);
            item.enclosure_type = std::move(type);
        }
    }
}
```

Reply:
Done.